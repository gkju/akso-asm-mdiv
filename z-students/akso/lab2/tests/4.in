1
2
5
23
235
1
24
2
4
7
9
6
4
3
2
2
15
20
10
100
90
80
70
60
50
40
30
20
10
0
-10
-20
-30
-40
-50
-60
-70
-80
-90
-100
