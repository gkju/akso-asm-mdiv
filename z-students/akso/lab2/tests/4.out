-100
-90
-80
-70
-60
-50
-40
-30
-20
-10
0
1
1
2
2
2
2
3
4
4
5
6
7
9
10
10
15
20
20
23
24
30
40
50
60
70
80
90
100
235
